### DISCUSSION TITLE
Hi, when I set up the individual model to show PK-analysis, AUC and Cmax can appear, but when I set up the group and click to show PK-analysis, the software reports an error, as follows, how should I modify it

### DISCUSSION THREAD

**Discussion by 2225110494** [Open-Systems-Pharmacology/Forum#1605] [Questions & Problems]
Hi, when I set up the individual model to show PK-analysis, AUC and Cmax can appear, but when I set up the group and click to show PK-analysis, the software reports an error, as follows, how should I modify it
![ddc3bb03f73cbd98e93f193e7846f0a](https://github.com/Open-Systems-Pharmacology/Forum/assets/143402499/563752c8-19c2-48f0-91b5-063e121bea9a)

**Comment by Yuri05** [comment]
From the error description, it seems that you have the same problem as [here](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/2632).
This issue was fixed, and the fix will be available in the upcoming version 12 of the OSP Suite.
In the meanwhile, you can try the workaround described [here](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/2632#issuecomment-1560694547)
